### PR TITLE
Clark/make io 301 redirects

### PR DESCRIPTION
**Pull request by clarkmcadoo**
This PR solves the splat redirects as well as our issue with having true 301 redirects

### CODE REVIEW

**Comment by clarkmcadoo** [comment]
This is testable at: https://developerwebsiteclarkmakeio301.gtsb.io/instant-observability

Screen Shot 2022-03-09 at 10 46 03 AM

**Comment by aswanson-nr** [comment]
Since this adds the splat redirects, could you also remove [this](https://github.com/newrelic/developer-website/blob/develop/gatsby-node.js#L101-L109) part of the graphql query and the `src/data/quickstarts.json` file?

**Comment by clarkmcadoo** [comment]
@aswanson-nr I was able to do most of your requests, but some of the plugins in the gatsby config still seem to be needed, the site wouldn't build without them.. Probably could audit this in the future and remove all the unnecessary ones in a different ticket one day?

File: gatsby-config.js [hunk 1]
```diff
@@ -167,13 +167,6 @@ module.exports = {
...
     },
-    {
-      resolve: 'gatsby-source-filesystem',
-      options: {
-        name: 'quickstarts',
-        path: `${__dirname}/src/data/quickstarts.json`,
-      },
-    },
     {
       resolve: 'gatsby-transformer-json',
```
Comment (aswanson-nr) [inline gatsby-config.js:171]: One last thing to remove! This transformer is only used for the quickstart json objects

**Review by aswanson-nr — DISMISSED** [review]

**Review by zstix — APPROVED** [review]

**Review by zstix — APPROVED** [review]

**Pull request merged**